### ISSUE TITLE
Stops working after turning off

### ISSUE THREAD

**Issue by Mayul** [DrNoLife/Destiny-2-Solo-Enabler#45]
After use, I have to delete the two destiny 2 inbound rules and then restart PC to be able to use solo enabler. The button is clickable and lights up but still says off.  I've tried every combination of settings but still the same results.

I'm on windows 11 v22H2 os build 22621.2134 if any of that matters.

![image](https://github.com/DrNoLife/Destiny-2-Solo-Enabler/assets/11929379/0cb78d5a-cbdc-47c1-b251-2665bb1341ac)

![image_2023-08-23_145056717](https://github.com/DrNoLife/Destiny-2-Solo-Enabler/assets/11929379/fc7a98be-9668-4a58-824f-5bda4e1a1f2e)

**Comment by BLTplayz** [comment]
This is an interesting case, sorry if this is repetitive but have you made sure it's being run with elevated privileges? Just some basic troubleshooting steps.

**Comment by Mayul** [comment]
Yeah, I have it set to run as admin both in shortcut and manually before and after computer restarts.

**Comment by Geist4587** [comment]
Please please watch my videos I don't want any followers I need the help of the masses Bungie the Destiny 2 developers have been hacking my HP Omen PC for over a year I've asked for help everywhere Facebook Instagram YouTube and Twitter put me in incognito so the truth  It doesn't come out, the sad thing is I haven't even paid it off completely and they don't stop hacking me, please distribute my videos, help me, I've been fighting Bungie for a long time with depression.  https://youtube.com/shorts/feRDkmBOyA4?si=GoWKFnyxqIi-eWkg
